src/data-types/date.ts

```diff
@@ -0,0 +1,58 @@
+import { DataType, ParameterData } from '../data-type';
+
+//This is to gain access to global objects in 'use strict' mode after typescript conversion to javascript. 
+var global = (0, eval)('this');
```

Comment:
I don't think this workaround is required. 😬 In Node.js, there's always a `global` variable defined, that allows accessing the global object. This `var` declaration here shadows the _real_ `global`. If you just remove this line, the other references of `global` (and `global.Date`) will still continue working. 👍

Reply:
As a bonus, your `global` variable has type `any`, while the proper `global` has a proper type and will allow proper typechecking for the usage of `global.Date`. 👍

Reply:
That's funny, I actually didn't realize 'global' was already defined in Node.js :sweat_smile: